DateBox: Add Native pickerType Notes (#8727)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateBox/1 Configuration/pickerType.md

    [note]

    When **pickerType** is *"native"*, DateBox behavior changes as follows:

>     - The component overrides [DevExtreme localization](/concepts/Common/Localization '/Documentation/Guide/Common/Localization/') settings and formats dates using locales of user devices.

Reviewer: ```suggestion
    - The component overrides [DevExtreme localization](/concepts/Common/Localization '/Documentation/Guide/Common/Localization/') settings and formats dates according to the user device locale.
```

>     - DateBox emits [onValueChanged](/api-reference/10%20UI%20Components/dxDateBox/1%20Configuration/onValueChanged.md '/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#onValueChanged') as users edit the value instead of when editing is finished (default).

Reviewer: ```suggestion
    - DateBox raises [onValueChanged](/api-reference/10%20UI%20Components/dxDateBox/1%20Configuration/onValueChanged.md '/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#onValueChanged') while the user edits the value, not after editing is complete.
```

>     - The component does not emit the following events:

Reviewer: ```suggestion
    - The component does not raise the following events:
```
